api/api.go

```diff
@@ -556,17 +555,76 @@ func ConfigQuery(cl *graphql.Client, configPath string, orgSlug string, params p
 		}
 		request.Var("pipelineParametersJson", string(pipelineParameters))
 	}
+
 	if orgSlug != "" {
 		request.Var("orgSlug", orgSlug)
 	}
+
 	request.SetToken(cl.Token)
 
 	err = cl.Run(request, &response)
-
 	if err != nil {
 		return nil, errors.Wrap(err, "Unable to validate config")
 	}
+	if len(response.BuildConfig.ConfigResponse.Errors) > 0 {
+		return nil, &response.BuildConfig.ConfigResponse.Errors
+	}
+
+	return &response.BuildConfig.ConfigResponse, nil
+}
 
+// ConfigQuery calls the GQL API to validate and process config with the org id
+func ConfigQuery(cl *graphql.Client, configPath string, orgId string, params pipeline.Parameters, values pipeline.Values) (*ConfigResponse, error) {
```

Comment:
Nitpick so optional, but I think we're capitalizing ID in the rest of the project, so orgID.